youtube2zim/scraper.py

```diff
@@ -285,7 +307,7 @@ def run(self):
             f"downloading all videos, subtitles and thumbnails (concurrency={self.max_concurrency})"
         )
         logger.info(f"  format: {self.video_format}")
-        logger.info(f"  recompress: {self.low_quality}")
+        logger.info(f"  quality: {self.video_quality}")
```

Comment:
you must add log informing that you'll use the cache.
Also indicate the netloc and the bucket name.

Reply:
I suggest not doing this here, because we need to recheck few things here

Reply:
what things? can you elaborate ?

Reply:
NVM